infra/app/adminweb.bicep

```diff
@@ -0,0 +1,47 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+param rgName string = ''
+param storageAccountName string = ''
+param formRecognizerName string = ''
+param contentSafetyName string = ''
+param allowedOrigins array = []
+param appServicePlanId string
+param appCommandLine string = 'python -m streamlit run Admin.py --server.port 8000 --server.address 0.0.0.0 --server.enableXsrfProtection false'
+param applicationInsightsName string = ''
+param keyVaultName string = ''
+param azureOpenAIName string = ''
+param azureCognitiveSearchName string = ''
+@secure()
+param appSettings object = {}
+param serviceName string = 'adminweb'
+param authType string
+
+module adminweb '../core/host/appservice.bicep' = {
+  name: '${name}-app-module'
+  params: {
+    name: name
+    location: location
+    tags: union(tags, { 'azd-service-name': serviceName })
+    allowedOrigins: allowedOrigins
+    appCommandLine: appCommandLine
+    applicationInsightsName: applicationInsightsName
+    appServicePlanId: appServicePlanId
+    appSettings: union(appSettings, {
+      AUTH_TYPE: authType
+      AZURE_OPENAI_KEY: listKeys(resourceId(subscription().subscriptionId, rgName, 'Microsoft.CognitiveServices/accounts', azureOpenAIName), '2023-05-01').key1
```

Comment:
Can you test something?  Can you try to get the resourceGroupName with 'resourceGroup().name"?  

I think the module is scoped to a resourceGroup which is set in the main.bicep file.  So, if the above works, then we may not have to pass in the rgName.

Reply:
Sounds great! We tried this and it worked fine. Updated.